gitlab/tests/objects/test_projects.py

```diff
@@ -9,6 +9,11 @@
 
 
 project_content = {"name": "name", "id": 1}
+import_content = {
+    "id": 1,
+    "name": "project",
+    "import_status": "scheduled",
+}
```

Comment:
Ahh good catch.  GitLab is returning a `201 Created` for a successful `POST` to https://gitlab.example.com/api/v4/import/bitbucket_server.  I'll update the `resp_import_bitbucket_server` mock to have a status code of 201.

The real response content has other fields also, which I ould add to the mock content, if you want; something like:

```
import_content = {
    "id": 1,
    "name": "some_project",
    "full_path": "some_group/some_project",
    "full_name": "Some Group / Some Project",
    "import_source": "https://some.bitbucketserver.com/projects/some_project/repos/some_repo",
    "import_status": "scheduled",
    "human_import_status_name": "scheduled",
    "provider_link": "https://some.bitbucketserver.com/projects/TER/repos/eits/browse",
}
```

More generally, is this unit test worth it?  It really only tests that some request gets `POST`ed to the correct GitLab API endpoint.

I'll also remove the test for the Github import, as I'm not actually sure of the real-world response, and thus I'm not sure how to mock it.

Reply:
Hm yeah, this would need a bitbucket container and a bunch of setup for proper integration tests. But I think it's still ok to have that 1 test hit, even if it's more basic asserts, as a sanity check in case something changes later in lower methods later :)